src/Service/View/ContextBuilder.php

```diff
@@ -177,6 +177,7 @@ private function getCommonContextContent(): array
             'accounts_user_id' => $this->accountsDataProvider->getAccountsUserId(),
             'accounts_shop_id' => $this->accountsDataProvider->getAccountsShopId(),
             'accounts_token' => $this->accountsDataProvider->getAccountsToken(),
+            'accounts_component_loaded' => false,
```

Comment:
Always to false ?

Reply:
The value is changed when the component is actually loaded in the template

https://github.com/PrestaShopCorp/ps_mbo/pull/670/files#diff-8abe5aa8fdd8f398c83db7c61455bf18bfa9c2af7b0669a3203684d66f239300R37